### PR TITLE
docs: fix typos

### PR DESCRIPTION
**Pull request by kianmeng**
found via `codespell -L copie,datas,pres,fo,tooks,noo,ue,ket,frop`

### CODE REVIEW

File: misc/check-twitter.sh [hunk 1]
```diff
@@ -46,7 +46,7 @@ check '2016-12-13 Tue 20:23.*TIL:.*pypi.python.org/pypi/coloredlogs'
 
 
 # https://twitter.com/karlicoss/status/472151454044917761
-# archive isn't explaning images by default
+# archive isn't explaining images by default
```
Comment (karlicoss) [inline misc/check-twitter.sh:49]: think that was meant to be "expanding"

File: my/core/kompress.py [hunk 2]
```diff
@@ -46,7 +46,7 @@ def _zstd_open(path: Path, *args, **kwargs) -> IO:
 # TODO use the 'dependent type' trick for return type?
 def kopen(path: PathIsh, *args, mode: str='rt', **kwargs) -> IO:
     # just in case, but I think this shouldn't be necessary anymore
-    # since when we cann .read_text, encoding is passed already
+    # since when we can .read_text, encoding is passed already
```
Comment (karlicoss) [inline my/core/kompress.py:49]: think that was meant to be "call"

File: my/photos/main.py [hunk 3]
```diff
@@ -43,7 +43,7 @@ def _basename(self) -> str:
             if self.path.startswith(bp):
                 return self.path[len(bp):]
         else:
-            raise RuntimeError(f'Weird path {self.path}, cant match against anything')
+            raise RuntimeError(f'Weird path {self.path}, can't match against anything')
```
Comment (karlicoss) [inline my/photos/main.py:46]: need to change outer quotes to double quotes, otherwise we get syntax error

**Review by karlicoss — CHANGES_REQUESTED** [review]
thanks! a few comments

**Comment by kianmeng** [comment]
:tada: :tada: :tada: :tada: :tada:

**Pull request merged**